### ISSUE TITLE
"Official" template project

### ISSUE THREAD

**Issue by clbarnes** [asciidoctor/asciidoctor#3430]
Whenever I reach for asciidoctor to format a document, I find myself struggling against the ruby ecosystem which I am not familiar with to do some basic tasks. It would be awesome to have a template project which contains a Gemfile, Rakefile and Guardfile, to allow anyone with a ruby installation access to features like easy build recipes (docbook, html, optionally PDF), file watching and so on. A Guard recipe watching asciidoc files *and* files in particular directories (for images, CSVs and so ons) would be excellent as part of that.

This would probably be trivial for someone familiar with the ecosystem, but that shouldn't be a dependency for convenient asciidoc workflows.

**Comment by clbarnes** [comment]
Here are my efforts for a project with this sort of setup: https://github.com/clbarnes/fiya-fudge

**Comment by mojavelinux** [comment]
I agree having a project template would be nice. In a certain context, that's what the Docker image is providing (https://github.com/asciidoctor/docker-asciidoctor). But I understand that not everyone is using Docker / podman as a starting point.

I don't have the bandwidth to pursue a template project at the moment. But I'm happy to promote community efforts that provide various starting points. If one really takes hold, we could find a place for it in the Asciidoctor ecosystem.
